rbac/processor/event_handler.py

```diff
@@ -119,8 +125,19 @@ def namespaces(self):
         return addresser.family.namespaces
 
     def apply(self, transaction, state):
-        payload = RBACPayload()
-        payload.ParseFromString(transaction.payload)
+        try:
```

Comment:
Docstrings on this whole file

Reply:
Isn't my code, this is legacy code that will soon disappear.

Reply:
cool if it is legacy this one is fine to not do docstrings